proto/testing/ssz_static_test.go

```diff
@@ -52,7 +51,13 @@ func runSSZStaticTests(t *testing.T, config string) {
 						return beaconState.HashTreeRoot(context.Background())
 					}
 				} else {
-					htr = ssz.HashTreeRoot
+					htr = func(s interface{}) ([32]byte, error) {
+						sszObj, ok := s.(fssz.HashRoot)
+						if !ok {
+							return [32]byte{}, errors.New("could not get hash root, not compatible object")
+						}
+						return sszObj.HashTreeRoot()
+					}
```

Comment:
Can this signature be changed for the argument to accept `fssz.HashRoot`? Compile time error would be better imo.

Reply:
I can't because the same signature is used for the beacon state HTR above this.